BibBuddy/app/src/androidTest/java/de/bibbuddy/NotesFragmentUITest.java

```diff
@@ -19,191 +22,173 @@
+  // TODO: Needs to be adjusted as soon as more note types are added
+  @Test
+  public void RecyclerViewListDisplay_Test() {
+    ViewActions.closeSoftKeyboard();
+    onView(withId(idText)).check(matches(isDisplayed()));
+    onView(withId(R.id.noteModDate)).check(matches(isDisplayed()));
+    onView(withId(R.id.noteName)).check(matches(isDisplayed()));
+    onView(withId(R.id.noteType)).check(matches(isDisplayed()));
+    new DrawableMatcher(R.drawable.document)
+        .matchesSafely(itemView);
+    // onView(withId(R.id.noteType)).check(matches(withId(R.drawable.microphone));
+    // onView(withId(R.id.noteType)).check(matches(withId(R.drawable.picture)));
+  }
+
+  // TODO: Needs to be adjusted as soon as more note types are added
+  @Test
+  public void OpenEditorOnItemClick_Test() {
+    ViewActions.closeSoftKeyboard();
+    onView(withId(idText)).perform(click());
+    onView(withId(R.id.fragment_text_note_editor)).check(matches(isDisplayed()));
+      /*
+      onView(withId(id2)).perform(click());
+      onView(withId(nF.getActivity().getSupportFragmentManager().findFragmentById(R.id.fragment_voice_note_editor).getId())).check(matches(isDisplayed()));
+      onView(withId(id3)).perform(click());
+      onView(withId(nF.getActivity().getSupportFragmentManager().findFragmentById(R.id.fragment_picture_note_editor).getId())).check(matches(isDisplayed()));
+      */
+  }
+
+  @Test
+  public void DeletePanelDisplay_Test() {
+    ViewActions.closeSoftKeyboard();
+    onView(withId(idText)).perform(longClick());
+    new DrawableMatcher(R.color.flirt_light).matchesSafely(itemView);
+    onView(withId(R.id.hidden_delete_panel)).check(matches(isDisplayed()));
+    new DrawableMatcher(R.color.alert_red)
+        .matchesSafely(itemView.getRootView().findViewById(R.id.panel_delete));
+    new DrawableMatcher(R.drawable.delete)
+        .matchesSafely(itemView.getRootView().findViewById(R.id.panel_delete));
+    onView(withId(R.id.hidden_delete_panel)).perform(click());
+    onView(withId(idText)).check(doesNotExist());
+  }
+
+  @Test
+  public void DeletePanelPerformDelete_Test() {
+    ViewActions.closeSoftKeyboard();
+    onView(withId(idText)).perform(longClick());
+    onView(withId(R.id.hidden_delete_panel)).check(matches(isDisplayed()));
+    onView(withId(idText)).perform(longClick());
+    onView(withId(R.id.hidden_delete_panel)).check(matches(not(isDisplayed())));
+  }
+
+
+  @Test
+  public void OnItemSwipeLeft_Test() {
+    onView(withId(idText)).perform(swipeLeft());
+    new DrawableMatcher(R.color.alert_red).matchesSafely(itemView);
+    new DrawableMatcher(R.drawable.delete).matchesSafely(itemView);
+  }
+
+  @Test
+  public void DeleteSnackbarDisplayOnSwipeLeft_Test() {
+    onView(withId(idText)).perform(swipeLeft());
+    onView(withText(R.string.delete_notification)).check(matches(isDisplayed()));
+    onView(withText(R.string.undo)).check(matches(isDisplayed()));
+  }
+
+  @Test
+  public void RestoreItemViewOnUndo_Test() {
```

Comment:
There is a checkstyle error:
`Method name 'RestoreItemViewOnUndo_Test' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.`